bashtop

```diff
@@ -425,6 +443,40 @@ quit_() { #? Clean exit
 	exit 0
 }
 
+update_ ()  {  #  update label  menu  to   upgrade  to new version   
+     
+    local -a  root_binary_path=("/bin/" "/usr/bin/")  
+    local  src=$(basename $0)
```

Comment:
Again will fail to find update if running version isn't named "bashtop"

Reply:
i' ll fix that